core/storage/statistics/gae_models.py

```diff
@@ -24,9 +24,9 @@
 from core.platform import models
 import feconf
 import utils
-
+# pylint: disable=relative-import
```

Comment:
This should not be here, it wasn't needed before. Might be a local issue with your machine if it's needed, but in that case it's the local issue that should be fixed.

Reply:
done!!